src/forms/forms.css

```diff
@@ -0,0 +1,114 @@
+.omrs-checkbox {
+  display: inline-block;
+  vertical-align: middle;
+  user-select: none;
+  cursor: pointer;
+}
+
+.omrs-checkbox input[type="checkbox"] {
+  opacity: 0;
+  position: absolute;
+}
+
+.omrs-checkbox label {
+  position: relative;
+  padding-left: 1.5rem;
+}
+
+.omrs-checkbox label::before,
+.omrs-checkbox label::after {
+  position: absolute;
+}
+
+.omrs-checkbox label::before {
+  content: "";
+  display: inline-flex;
+  height: 1rem;
+  width: 1rem;
+  border: 0.125rem solid var(--omrs-color-ink-medium-contrast);
+  top: 0rem;
+  left: 0rem;
+}
+
+.omrs-checkbox input[type="checkbox"]:checked + label::before {
+  border: 0.125rem solid var(--omrs-color-interaction);
+  background: var(--omrs-color-interaction);
+}
+
+.omrs-checkbox label::after {
+  content: none;
+  display: inline-block;
+  height: 0.25rem;
+  width: 0.75rem;
+  transform: translate(-45%, -40%) rotate(-50deg);
+  left: 0.5625rem;
+  top: 0.5rem;
+}
+
+.omrs-checkbox input[type="checkbox"]:checked + label::after {
+  content: "";
+  border-left: 0.125rem solid var(--omrs-color-ink-white);
+  border-bottom: 0.125rem solid var(--omrs-color-ink-white);
+}
+
+.omrs-checkbox input[type="checkbox"]:checked + label {
+  color: var(--omrs-color-interaction);
+}
+
+/* Radio Button styles */
+.omrs-radio-button {
+  display: inline-block;
+  vertical-align: middle;
+  user-select: none;
+  cursor: pointer;
+}
+
+.omrs-radio-button input[type="radio"] {
+  opacity: 0;
+  position: absolute;
```

Comment:
Same here about positioning elements without knowing what the positioned ancestor will be